pkg/process/checks/net_test.go

```diff
@@ -125,8 +116,8 @@ func TestNetworkConnectionBatchingWithDNS(t *testing.T) {
 	for i, c := range chunks {
 		connections := c.(*model.CollectorConnections)
 
-		// Only the first chunk should have a DNS mapping!
-		if i == 0 {
+		// Only the last chunk should have a DNS mapping
```

Comment:
What is the reason for this change? From the code, it seems only `p[0]` has a remote address with an associated DNS entry.

Reply:
When the connections are batched they are sorted by `rAddr`, and since the other `rAddrs` are simply `""` the connection with `rAddr = "1.1.2.3"` ends up at the end of the list